gyp_xwalk

```diff
@@ -351,6 +351,10 @@ if __name__ == '__main__':
     args.append('-Dnotifications=1')
     args.append('-Drelease_unwind_tables=0')
 
+  if gyp_vars_dict.get('OS') == 'win':
+    args.append('-Denable_widevine=1')
+    args.append('-Denable_pepper_cdms=1')
```

Comment:
The video not playing issue is caused by MP4 codec, does not mean the windwine not working. You either find a webm widewine test case, or offer special build for verification.

BTW, isn't `-Denable_widevine=1 -Denable_pepper_cdms=1` need for android build?